daisy_workflows/linux_common/utils/diskutils.py

```diff
@@ -56,6 +56,10 @@ def MountDisk(disk):
   # mounting the filesystems in the correct order.
   mps = g.inspect_get_mountpoints(roots[0])
 
+  g.gcp_image_distro = g.inspect_get_distro(roots[0])
+  g.gcp_image_major = str(g.inspect_get_major_version(roots[0]))
+  g.gcp_image_minor = str(g.inspect_get_minor_version(roots[0]))
```

Comment:
Do all distributions have major/minor versions?

Reply:
Zero is returned from `inspect_get_major_version` and `inspect_get_minor_version` if those fields aren't found. For sanity testing, I ran the ovf import tests using this code and there were no errors.

Here's what the python code is calling:
  http://libguestfs.org/guestfs.3.html#guestfs_inspect_get_major_version